packages/core/src/button/base-button.element.scss

```diff
@@ -48,10 +48,7 @@
   user-select: none;
   vertical-align: middle;
   white-space: nowrap;
-
-  &:active {
-    box-shadow: 0 #{$cds-global-space-1} 0 0 var(--box-shadow-color) inset;
-  }
+  width: 100%;
```

Comment:
This fixes the bug of the button not filling its space when displayed as a flex child. When using layout or flexbox and stretching the host would stretch but the private-host element did not fill its space correctly.